real_time/surface_src/java_src/Alice/build.gradle

```diff
@@ -11,6 +11,9 @@ dependencies {
     testCompile 'junit:junit:4.12'
 
     compile project (':BuggyRos')
+    //for jason object extraction from strings 
```

Comment:
prefer "json", which is javascript object notation, to "jason", who is my younger brother.